app/src/db/models/permit.ts

```diff
@@ -54,12 +60,24 @@ export default {
     };
   },
 
-  fromPrismaModelWithNotes(input: PrismaGraphPermitNotes): Permit {
-    const permit = this.fromPrismaModel(input);
+  fromPrismaModelWithNotesTracking(input: PrismaGraphPermitNotesTracking): Permit {
+    const permit = this.fromPrismaModelWithTracking(input);
     if (permit && input.permit_note) {
       permit.permitNote = input.permit_note.map((note) => permit_note.fromPrismaModel(note));
     }
 
+    return permit;
+  },
+
+  fromPrismaModelWithTracking(input: PrismaGraphPermitTracking): Permit {
```

Comment:
if full map done this shouldn't be required I believe, and then above "from" function would still be just defined as `fromPrismaModelWithNotes`